src/main/java/uk/gov/companieshouse/pscstatement/delta/mapper/MapperUtils.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.companieshouse.pscstatement.delta.mapper;
+
+import java.util.Base64;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+public class MapperUtils {
+    /**
+     * encode the String passed in for use in links and ids.
+     */
+    public static String encode(String unencodedString) {
+        String salt = "ks734s_sdgOc4£b2";
```

Comment:
Could this be changed to a config value instead of being hardcoded. Makes it more secure and easier for FESS if the salt ever changes